gallery/src/handleQueryParams.ts

```diff
@@ -27,4 +27,11 @@ export default function handleQueryParams(queryParams: ParsedQuery) {
   else {
     classList.remove('gallery-hide-sidebar');
   }
+
+  if (queryParams.showGlobalSidebar === 'true') {
```

Comment:
What's the purpose of this?  We have some pages that have the global sidebar and some that don't, so to additionally be able to hide it on pages that have it seems redundant

Reply:
> What's the purpose of this? We have some pages that have the global sidebar and some that don't, so to additionally be able to hide it on pages that have it seems redundant

It's old and should have been removed: https://github.com/sonatype/sonatype-react-shared-components/pull/371/commits/2e9380a7be441078ceb31f7af2a5269f3bca2c2b